### DISCUSSION TITLE
AndroidTV support

### DISCUSSION THREAD

**Discussion by dystopiaTD** [bttv-android/bttv#380] [Ideas]
I just wanted to ask whether there are any plans to make this work on android TVs.

I'd greatly appreciate it :]

**Comment by FoseFx** [comment]
As I said [here](https://github.com/bttv-android/bttv/issues/340#issuecomment-1003561717) before there are no plans to mod the Android TV Twitch App as well.
I unfortunately do not have the time to do this right now :/

**Reply by MarkusKlemela** [reply]
The app loads just fine on my Android TV but I can't do anything. Wouldn't it theoretically be usable just by adding support for remote control navigation?
